foia_hub/api.py

```diff
@@ -12,6 +11,56 @@
 
 from foia_hub.models import Agency, Office, Requester, FOIARequest
 
+import re
+import string
+
+
+def sanitize_search_term(term):
+    # Replace all puncuation with spaces.
+    allowed_punctuation = set(['&', '|', '"', "'"])
+    all_punctuation = set(string.punctuation)
+    punctuation = "".join(all_punctuation - allowed_punctuation)
+    term = re.sub(r"[{}]+".format(re.escape(punctuation)), " ", term)
+
+    # Substitute all double quotes to single quotes.
+    term = term.replace('"', "'")
+    term = re.sub(r"[']+", "'", term)
+
+    # Create regex to find strings within quotes.
+    quoted_strings_re = re.compile(r"('[^']*')")
+    space_between_words_re = re.compile(r'([^ &|])[ ]+([^ &|])')
+    spaces_surrounding_letter_re = re.compile(r'[ ]+([^ &|])[ ]+')
+    multiple_operator_re = re.compile(r"[ &]+(&|\|)[ &]+")
+
+    tokens = quoted_strings_re.split(term)
+    processed_tokens = []
+    for token in tokens:
+        # Remove all surrounding whitespace.
+        token = token.strip()
+
+        if token in ['', "'"]:
+            continue
+
+        if token[0] != "'":
+            # Surround single letters with &'s
+            token = spaces_surrounding_letter_re.sub(r' & \1 & ', token)
+
+            # Specify '&' between words that have neither | or & specified.
+            token = space_between_words_re.sub(r'\1 & \2', token)
```

Comment:
Does this mean that if I search for 'health education' it will treat those as an 'and' and not an 'or'? I'm actually not clear - is that how we would expect it to behave?